src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/linked_orders.html.twig

```diff
@@ -0,0 +1,75 @@
+{#**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="card" id="linkedOrdersPanel">
+  <div class="card-header">
+    <h3 class="card-header-title">
+      {{ 'Linked orders'|trans({}, 'Admin.Orderscustomers.Feature') }}
+      <span class="badge badge-primary rounded">{{ orderForViewing.linkedOrders.linkedOrders|length }}</span>
+    </h3>
+  </div>
+
+  <div class="card-body">
+    <table class="table mb-3">
+      <thead>
+      <tr>
+        <th>
+          {{ 'Order ID'|trans({}, 'Admin.Orderscustomers.Feature') }}
+        </th>
+        <th>
+          {{ 'Status'|trans({}, 'Admin.Global') }}
+        </th>
+        <th>
+          {{ 'Amount'|trans({}, 'Admin.Global') }}
+        </th>
+        <th></th>
+      </tr>
+      </thead>
+      <tbody>
+      {% if orderForViewing.linkedOrders.linkedOrders is not empty %}
```

Comment:
Shouldn't the condition be on the whole block? Because this will display an empty table
Or at least display an empty state?

Reply:
This block is only included when this array is not empty. See https://github.com/PrestaShop/PrestaShop/pull/19704/files#diff-57115db6da7baa5aa2cc23cd7d5a5f6aR97
````
 {% if orderForViewing.linkedOrders.linkedOrders is not empty %}
      {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/linked_orders.html.twig' %}
    {% endif %}
```

I repeated the condition inside the block "just in case" someone extends the BO theme and include it anyway although it is empty.
It's a developer reflex 😁 : "always check array is empty before using foreach"